Updated North Sámi translations in the cookie files

nikopartanen opened the pull request:
We noticed there were some missing Sámi translations in the cookie setting section, and those are now all checked and translated.

nikopartanen committed:
Create se.ini

I added a few more translations and created this file. The work is on-going, as I need a few more translations.

nikopartanen committed:
Create se.ini

I add some of the translations, but we clearly have a few more that are missing.

nikopartanen committed:
Merge branch 'NatLibFi:dev' into patch-4

nikopartanen committed:
Checked translations from Mari Guttorm

Mari Guttorm checked these translations, and I updated them now

nikopartanen committed:
New checked translations

Mari Guttorm checked these and added some new translations about cookies. I added these all now into these files.

EreMaijala commented on languages/CookieConsent/se.ini:
```diff
@@ -0,0 +1,32 @@
+Accept All Cookies = "Dohkket buot niesttážiid" 
```
This file should be in the upstream vufind version. You can add these with Lokalise or with a pull request to vufind-org/vufind.

EreMaijala replied:
Or I can create the pull request on your behalf if you wish so.

nikopartanen replied:
I added this now, Lokalise seems to be working very nicely

nikopartanen commented:
Should I delete that line from the North Sámi translation file?

EreMaijala commented:
As discussed in Slack, everything in languages needs to go upstream, so Lokalise. You'll need to remove the whole file from this pull request.

nikopartanen committed:
Added a missing translation

There was one translation that was still missing, added now

nikopartanen committed:
Delete languages/CookieConsent/se.ini

I removed this file now and brought its content to Lokalize.

nikopartanen commented:
I added now the content of that file into Lokalize and removed the file in GitHub within this pull request, please let me know if anything else should be corrected.
